runelite-client/src/main/java/net/runelite/client/plugins/specialcounter/SpecialCounterPlugin.java

```diff
@@ -66,6 +68,7 @@
 	private int specialPercentage;
 	private Actor lastSpecTarget;
 	private int lastSpecTick;
+	private final int[] npcWhitelist = {NpcID.ZOMBIFIED_SPAWN_8063, NpcID.ZOMBIFIED_SPAWN, NpcID.DARK_ENERGY_CORE};
```

Comment:
I'd rather this be a property of `Boss` so we don't have some special case in the plugin that's not apparent when reading the enum.